File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int n=nums.length;
        int cnt=0;
        int max_ones=0;
        for(int i=0;i<n;i++){
            if(nums[i]==1){
                cnt++;
            }
            else{
                max_ones = Math.max(cnt,max_ones);
                System.out.println(max_ones);
                cnt=0;
            }
        }

        max_ones = Math.max(cnt,max_ones);

        return max_ones;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int i=0,j=0,max=0;
        for(j=0;j<nums.size();j++){
            if(nums[j]!=1){
                if(j-i > max){
                    max = j-i;
                }
                i=j+1;
            }
        }
        if(max < j-i){
            max = j-i;
        }
        return max;
    }
};